cmd/run/run.go

```diff
@@ -316,8 +316,8 @@ func MustDefaultConfigWithRandomPorts() *Config {
 	defer l.Close()
 	grpcPort := l.Addr().(*net.TCPAddr).Port
 
-	config.GRPC.Addr = fmt.Sprintf("0.0.0.0:%d", grpcPort)
-	config.HTTP.Addr = fmt.Sprintf("0.0.0.0:%d", httpPort)
+	config.GRPC.Addr = fmt.Sprintf("localhost:%d", grpcPort)
```

Comment:
TLS requires localhost (not 0.0.0.0 ) because it doesn’t contain any IP SANs .  Thus, we will either have to specifically change TLS case to use localhost or hard code those cases.

Reply:
We may also want to consider https://github.com/openfga/openfga/issues/639 and update the default to localhost as well (or address this in a separate PR)

Reply:
We're talking about unrelated things here. In the scope of this PR we're not changing 0.0.0.0 to `localhost` to address TLS specific issues. If we want to do that, as part of work to address #639 for example, then let's save that for a separate PR. The purpose of this work is to avoid hard-coded ports. So let's keep it focused to that.